### PR TITLE
Update minkowski_difference.jl

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

File: src/ConcreteOperations/minkowski_difference.jl [hunk 1]
```diff
@@ -27,18 +27,18 @@ This function implements Theorem 2.3 in [1], which we state next.
 
 Suppose ``P`` is a polyhedron
 ```math
-P = \\{z ∈ ℝⁿ: sᵢᵀz ≤ rᵢ, i=1,...,N\\}.
+P = \\{z ∈ \\mathbb{R}^n: sᵢᵀz ≤ rᵢ,~i = 1 , \\ldots, N\\}.
```
Comment (schillic) [inline src/ConcreteOperations/minkowski_difference.jl]:
```suggestion
P = \\{z ∈ \\mathbb{R}^n: sᵢᵀz ≤ rᵢ,~i = 1, …, N\\}.
```
Does the `ℝ` not work?

Comment (mforets) [reply src/ConcreteOperations/minkowski_difference.jl]: In principle it is ok, however i saw that the superscript `n` is not in the right place, see: https://juliareach.github.io/LazySets.jl/latest/lib/binary_functions/#LazySets.minkowski_difference-Union{Tuple{N},%20Tuple{LazySet{N},LazySet{N}}}%20where%20N%3C:Real

Comment (mforets) [reply src/ConcreteOperations/minkowski_difference.jl]: It is too much below the R :laughing:  compare to: https://juliareach.github.io/LazySets.jl/latest/man/polyhedral_approximations/#Preliminaries-1

Comment (schillic) [reply src/ConcreteOperations/minkowski_difference.jl]: But then maybe just do the `^n`.

Comment (mforets) [reply src/ConcreteOperations/minkowski_difference.jl]: I can try if doing the `^n` makes a difference. But how do i get the `\mathbb{R}` on the first place in the Julia's REPL?

Comment (schillic) [reply src/ConcreteOperations/minkowski_difference.jl]: `\bbR<TAB>` (from [here](https://discourse.julialang.org/t/changes-to-latex-shortcuts-in-repl/9110))

Comment (mforets) [reply src/ConcreteOperations/minkowski_difference.jl]: :100:  Now if i do  `\bbR<TAB>\^n<TAB>` it is printed in the correct place in the REPL, however, when i print it here, again the `n` is a bit off, see: `julia> ℝⁿ`.

Comment (schillic) [reply src/ConcreteOperations/minkowski_difference.jl]: I would not consider the markdown rendering here. What does the html output look like? If my impression is right and `\\mathbb{R}` and `ℝ` render as the same symbol, then they should not influence the `n`.

File: src/ConcreteOperations/minkowski_difference.jl [hunk 2]
```diff
@@ -27,23 +27,23 @@ This function implements Theorem 2.3 in [1], which we state next.
 
 Suppose ``P`` is a polyhedron
 ```math
-P = \\{z ∈ ℝⁿ: sᵢᵀz ≤ rᵢ, i=1,...,N\\}.
+P = \\{z ∈ ℝ^n: sᵢᵀz ≤ rᵢ,~i = 1 , …, N\\}.
```
Comment (schillic) [inline src/ConcreteOperations/minkowski_difference.jl]:
```suggestion
P = \\{z ∈ ℝ^n: sᵢᵀz ≤ rᵢ,~i = 1, …, N\\}.
```

**Pull request merged**